docs/sources/datasources/_index.md

```diff
@@ -8,12 +8,14 @@ weight: 60
 
 # Data sources
 
-Grafana can query and integrate with many different types of databases. This is done by adding a **data source** of the type you want to query or integrate with.
-When you have created and configured a data source you are ready to start exploring and visualizing data, either in Explore or in a new Dashboard. A dashboard is composed of [panels]({{< relref "../panels-visualizations/" >}}), each panel contains a set of queries to one or more data sources.
+Grafana has built-in support for a wide range of _data sources_.
+These data sources might be files, services, or databases.
+You can also extend the supported data sources through one of the many [Plugins](/grafana/plugins/) (or build your own).
```

Comment:
Not sure how we usually talk about them - `supported` vs `built-in`

Reply:
Yeah, that might be a little bit of language from k6 docs. I wanted to avoid jargon like "native" or "out of the box".

I think we can avoid this issue entirely by just using _support_ as a verb.

> Grafana supports a wide range of _data sources_.

Shorter sentence, too :-) .

Reply:
It's also consistent, since it's the exact language of https://grafana.com/tutorials/build-a-data-source-plugin/

Reply:
[54e3654](https://github.com/grafana/grafana/pull/61639/commits/54e3654f2224c8491ad3cc6a031520d8f0a950c2)